plugin/src/main/java/io/snyk/languageserver/protocolextension/messageObjects/scanResults/Issue.java

```diff
@@ -0,0 +1,96 @@
+package io.snyk.languageserver.protocolextension.messageObjects.scanResults;
+
+public class Issue {
```

Comment:
I think this class needs `equals` and `hashCode` methods to be able to sort & compare them